tests/pipeline/test_engine.py

```diff
@@ -796,27 +784,30 @@ def test_drawdown(self):
         assert_frame_equal(expected, result)
 
 
-class ParameterizedFactorTestCase(TestCase):
+class ParameterizedFactorTestCase(WithTradingEnvironment, ZiplineTestCase):
+    sids = Int64Index([1, 2, 3])
+
+    @classmethod
+    def make_equity_info(cls):
+        cls.equity_info = ret = make_simple_equity_info(
```

Comment:
Almost all of the usages of `make_equity_info` are calling `make_simple_equity_info` with a fixed set of sids and a fixed start/end date.  Thoughts on making that the default with configurable ALL_CAPS args to `WithAssetFinder`?